backend/app/services/vectordb.py

```
@@ -0,0 +1,23 @@
+# app/vectordb.py
+import chromadb
+from chromadb.config import Settings
+
+client = chromadb.Client(Settings(
+    chroma_db_impl="duckdb+parquet",
+    persist_directory="./chroma_store"
+))
+
+collection = client.get_or_create_collection("resumes")
```

Add error handling for ChromaDB initialization and make configuration configurable.

The global initialization could fail silently, and configuration should be externalized.